docs/getting_started.md

```diff
@@ -18,7 +18,8 @@ chmod +x /usr/local/bin/argocd
 
 ## 2. Install ArgoCD
 ```
-argocd install
+cd manifests
+kubectl apply -f manifests/. --namespace argocd
```

Comment:
As discussed, lets not change getting_started.md until we cut a release that people can follow. You can create a different issue/PR for updating getting_started.md once we have 0.6